azure-iot-device/samples/async-hub-scenarios/provision_symmetric_key.py

```diff
@@ -27,6 +32,34 @@ async def main():
     print("The complete registration result is")
     print(registration_result.registration_state)
 
+    if registration_result.status == "assigned":
+        print("Will send telemetry from the provisioned device")
+        device_client = IoTHubDeviceClient.create_from_symmetric_key(
+            symmetric_key=symmetric_key,
+            hostname=registration_result.registration_state.assigned_hub,
+            device_id=registration_result.registration_state.device_id,
+        )
+        # Connect the client.
+        await device_client.connect()
+
+        async def send_test_message(i):
+            print("sending message #" + str(i))
+            msg = Message("test wind speed " + str(i))
+            msg.message_id = uuid.uuid4()
+            msg.correlation_id = "correlation-1234"
+            msg.custom_properties["count"] = i
+            msg.custom_properties["tornado-warning"] = "yes"
+            await device_client.send_message(msg)
```

Comment:
You don't need to set properties here (and other places), we want to make it as concise as possible